docs/design-proposals/arch/daemonset.md

```diff
@@ -2,6 +2,8 @@
 
 Due to the nature Arktos being a multiple-tenanted system, DaemonSet is only allowed in system tenant; only cluster admin is permitted to create/delete/update DaemonSet resources.
 
+Multiple-TP also causes similar issue to Kube-proxy; changes to kube-proxy is covered in this design.
+
 ### What is not covered yet
 At this stage of Arktos, PVC/PV for storage volume is not fully supported - out of scope of this DaemonSet design.
```

Comment:
any reason PV/PVC is related with this design?

Reply:
current arktos does npt support PV/PVC in multi-tenant yet.